airlock/business_logic.py

```diff
@@ -864,15 +864,15 @@ def get_request_file_status(
         rfile = self.get_request_file_from_urlpath(relpath)
         phase = self.get_turn_phase()
         decision = RequestFileDecision.INCOMPLETE
-        is_output_checker = user.output_checker and self.author != user.username
+        can_review = self.user_can_review(user)
 
         match phase:
             case ReviewTurnPhase.INDEPENDENT:
                 # already set - no one knows the current status
                 pass
             case ReviewTurnPhase.CONSOLIDATING:
                 # only output-checkers know the current status
```

Comment:
should the comment match code? my wording is awful though
```suggestion
                # only users that can review can see the current status
```